sink/sink

```diff
@@ -549,6 +549,13 @@ def check_prune(config, logs, dry_run):
                 sys.exit(0)
 
 
+def strip_secrets(line):
+    # Strip out GitHub-style secrets
+    line = re.sub(b'gh[a-z]_[0-9A-Za-z]*', b'gh*_***', line)
```

Comment:
This could have been `{5,}` instead of  `*` to not filter out trivial stuff like "gha_", but we can do with that for the time being.